gradle.properties

```diff
@@ -26,4 +26,4 @@ systemProp.org.gradle.warning.mode=fail
 systemProp.jdk.tls.client.protocols=TLSv1.2
 
 # jvm args for faster test execution by default
-systemProp.tests.jvm.argline=-XX:TieredStopAtLevel=1 -XX:ReservedCodeCacheSize=64m -Djava.security.manager=allow
+systemProp.tests.jvm.argline=-XX:TieredStopAtLevel=1 -XX:ReservedCodeCacheSize=64m
```

Comment:
@dblock reflection on this change, it works fine for JDK17 (our default build) and above but it turns out for JDK-8 to JDK-16, it actually hurts and leads to creation of `SecurityManager` with class name `allow`:

```
Error occurred during initialization of VM                                                                                                                                                                         
java.lang.Error: Could not create SecurityManager                                                                                                                                                                  
        at java.lang.System.initPhase3(java.base@11.0.12/System.java:2065)                                                                                                                                         
Caused by: java.lang.ClassNotFoundException: allow                                                                                                                                                                 
        at jdk.internal.loader.BuiltinClassLoader.loadClass(java.base@11.0.12/BuiltinClassLoader.java:581)                                                                                                         
        at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(java.base@11.0.12/ClassLoaders.java:178)                                                                                                      
        at java.lang.ClassLoader.loadClass(java.base@11.0.12/ClassLoader.java:522)                  
        at java.lang.Class.forName0(java.base@11.0.12/Native Method)                                     
        at java.lang.Class.forName(java.base@11.0.12/Class.java:398)                                                                                                                                               
        at java.lang.System.initPhase3(java.base@11.0.12/System.java:2050)
```

It does not break anything  at the moment, but future proof solution would be to apply `-Djava.security.manager=allow` only for JDK-18 and above.